src/pencilkit.cs

```diff
@@ -0,0 +1,243 @@
+//
+// PencilKit C# bindings
+//
+// Authors:
+//	TJ Lambert  <t-anlamb@microsoft.com>
+//
+// Copyright 2019 Microsoft Corporation All rights reserved.
+//
+
+#if MONOMAC
+using AppKit;
+using UIColor = AppKit.NSColor;
+using UIImage = AppKit.NSImage;
+
+using UIScrollViewDelegate = Foundation.NSObjectProtocol;
+using UIScrollView = Foundation.NSObject;
+using UIGestureRecognizer = Foundation.NSObject;
+using UIResponder = Foundation.NSObject;
+using UIView = Foundation.NSObject;
+using UIWindow = Foundation.NSObject;
+#else
+using UIKit;
```

Comment:
I'm confused re this #if/#else macro - is UIKit only available on non-mac devices?

Reply:
@whitneyschmidt correct, UIKit is not available on macOS, only iOS, tvOS and watchOS (on macOS there's AppKit instead, which is much older than UIKit).

Reply:
thanks @rolfbjarne!